core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/ccm/upgrade/UpgradeCcmService.java

```diff
@@ -145,6 +150,15 @@ public void updateTunnel(Long stackId) {
         stackService.setTunnelByStackId(stackId, Tunnel.latestUpgradeTarget());
     }
 
+    public void pushSaltState(Long stackId, Long clusterId) {
+        InMemoryStateStore.putStack(stackId, PollGroup.POLLABLE);
+        if (clusterId != null) {
+            InMemoryStateStore.putCluster(clusterId, PollGroup.POLLABLE);
+        }
```

Comment:
why is this necessary? usually stack state changes does the same

Reply:
I haven't asked to remove and retest. Although I see now it was moved from another class, which was deleted. I just don't understand why is it necessary at all.